pmlc/dialect/pxa/transforms/passes.h

```diff
@@ -15,6 +15,8 @@ namespace pmlc::dialect::pxa {
 
 std::unique_ptr<mlir::Pass> createAutoTileExamplePass();
 
+std::unique_ptr<mlir::Pass> createVectorizeExamplePass();
```

Comment:
Drop the `Example` from the name.

Reply:
This pass is purposely unsophisticated as the goal is really the transform.  We can expand the pass to be more general, but I think that should be in a separate PR.  The naming is done to suggest this.